lib/pages/campaign_form_page.dart

```diff
@@ -45,6 +54,10 @@ class _CampaignFormPageState extends State<CampaignFormPage> {
   }
 
   Future<void> saveCampaign() async {
+    setState(() {
+      loading = true;
+      error = "";
+    });
```

Comment:
a boolean variable should be prefixed with is, are. so isLoading

error? is it an error message? an error object?

Reply:
It should be errorMsg, but we use it in several other places. I'm leaving it for consistancy.

Reply:
You know what, it's only used withen forms, I think we can update it.